lib/tag/vdom.js

```diff
@@ -38,10 +38,11 @@ function mountTo(root, tagName, opts) {
 
 }
 
-riot.tag = function(name, html, css, fn) {
+riot.tag = function(name, html, css, attrs, fn) {
+  if (typeof attrs == 'function') {fn = attrs; attrs=css}
```

Comment:
@cognitom , I made your change, but I get `expected '' to equal 'single-quote'` in the browser test.

the < v2.0.10, if css is a function, attrs will be null, so that should not make a difference
the < v2.0.16 makes sense, but it gives me the error above.

What error do you get and do you have a test case we could add?